### DISCUSSION TITLE
How to group by table in duckDB when I set temp_directory = '/tmp/duckdb-tmp'

### DISCUSSION THREAD

**Discussion by thrmagic** [duckdb/duckdb#13366] [Q&A]
I have a parquet file(78mb),and create a table in duckDB,but throw error Out of Memory,so i  `set temp_directory = '/tmp/duckdb-tmp'` .but throw error `tried to read past the end`when i use `group by`

`c.query("SET temp_directory = '/tmp/duckdb-tmp'")`

`await c.query('CREATE TABLE dm_machine_part_recommendation_ap AS SELECT * FROM read_parquet(\'data.parquet\')')`

`const result = await c.query('select region,sub_region,segment,count(1) from "dm_machine_part_recommendation_ap" group by region,sub_region,segment order by region desc')`

![image](https://github.com/user-attachments/assets/e318be58-cd20-4735-ac51-c2c0573aab25)

**Comment by Tishj** [comment]
That looks like a duckdb-wasm specific error, perhaps you can raise a bug report over at <https://github.com/duckdb/duckdb-wasm>
